youtube_dl_gui/locale/ja_JP/LC_MESSAGES/youtube_dl_gui.po

```diff
@@ -0,0 +1,588 @@
+# Youtube-dlG localization file.
+# FIRST AUTHOR: Sotiris Papadopoulos <ytubedlg@gmail.com>, 2015.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: youtube-dlg 0.4\n"
```

Comment:
Why did you remove some headers like:
* POT-Creation-Date
* PO-Revision-Date
* Last-Translator

We need those for maintenance reasons. Please add them back.